tests/acceptance/tests/Forms/ValidateCaptchaInContactForm.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { test } from '@fixtures/AcceptanceTest';
+import { expect } from '@playwright/test';
+
+// Annotate entire file as serial run.
+test.describe.configure({ mode: 'serial' });
+
+test(
+    'As a customer, I expect to see and use a basic captcha function on the contact form.',
+    { tag: '@form @contact' },
+    async ({ ShopCustomer, StorefrontHome, StorefrontContactForm, DefaultSalesChannel, TestDataService }) => {
+
+        await TestDataService.setSystemConfig({'core.basicInformation.activeCaptchasV2': {'basicCaptcha': { 'name': 'basicCaptcha', 'isActive': true }} });
+
+        await test.step('Open the contact form modal on home page.', async () => {
+            await ShopCustomer.goesTo(StorefrontHome.url());
+            await StorefrontHome.contactFormLink.click();
+            await ShopCustomer.expects(StorefrontContactForm.cardTitle).toContainText('Contact');
+        });
+
+        await test.step('Fill out all necessary contact information.', async () => {
+            await StorefrontContactForm.salutationSelect.selectOption('Mr.');
+            await StorefrontContactForm.firstNameInput.fill('John');
+            await StorefrontContactForm.lastNameInput.fill('Doe');
+            await StorefrontContactForm.emailInput.fill('mail@test.com');
+            await StorefrontContactForm.phoneInput.fill('0123456789');
+            await StorefrontContactForm.subjectInput.fill('Test: Product question');
+            await StorefrontContactForm.commentInput.fill('Test: Hello, I have a question about your products.');
+            await StorefrontContactForm.basicCaptchaInput.fill('1234');
+            await StorefrontContactForm.privacyPolicyCheckbox.click();
+        });
+
+        await test.step('Validate the basic captcha is available.', async () => {
+            await ShopCustomer.expects(StorefrontContactForm.basicCaptcha).toBeVisible();
+            await ShopCustomer.expects(StorefrontContactForm.basicCaptchaImage).toBeVisible();
+            await ShopCustomer.expects(StorefrontContactForm.basicCaptchaRefreshButton).toBeVisible();
+        });
+
+        await test.step('Send and validate the contact form.', async () => {
```

Comment:
I would add some thing like:
```suggestion
        await test.step('Send and validate the contact form is submit successfully (or not sent).', async () => {
```

Reply:
adjusted the desciption